docs/joss_paper/paper.md

```diff
@@ -82,35 +82,35 @@ The loss functions described in the previous sections are often combined with a
 The latest result from a different registration formulation challenged the use of deformation regularisation and consequently the necessity of predicting dense displacement field [@hu2019conditional], also implemented in DeepReg.
 
 # Implemetation
-## Model
-A pair of moving image and fixed image is the input of the network. When available, pairs of moving labels and/or fixed labels can also be the input of the network, regardless of whether they are used to compute the loss. The network predicts a dense displacement field (DDF). Options are also available for the DDF being integrated by a predicted dense velocity field (DFV) or computed from a predicted affine transformation. For conditional segmentation, the network directly predicts warped moving labels [@hu2019conditional].
+## Registration network
+A pair of moving image and fixed image is the input of the network. When available, pairs of moving labels and/or fixed labels can also be the input of the network to compute the label dissimilarity loss. The network predicts a dense displacement field (DDF). Options are also available for the DDF being integrated by a predicted dense static velocity field, often as SVF or DVF, or computed from a predicted affine transformation. For conditional segmentation, the network directly predicts warped moving labels [@hu2019conditional].
```

Comment:
(Reads awkwardly) A pair of moving image and fixed image is the input of the network -> The network accepts two images as input: a fixed image, and a moving image.

Do we want to elaborate on what the fixed/moving image are - or is the knowledge of this assumed? It's not previously mentioned.